src/scanner.c

```diff
@@ -845,7 +845,6 @@ static void GetChar(void)
 {
   /* skip '\''                                                           */
   Char c = GET_NEXT_CHAR();
-
```

Comment:
?

Reply:
I often add a printf there to see where GAP has got to when something bad happens during startup. Did that to find `last := "2b defined"`, then haven't cleaned up properly :)